Sources/Subprocess/Configuration.swift

```diff
@@ -822,27 +801,51 @@ internal enum StringOrRawBytes: Sendable, Hashable {
     }
 }
 
-/// A simple wrapper on `FileDescriptor` plus a flag indicating
-/// whether it should be closed automactially when done.
-internal struct TrackedFileDescriptor: Hashable {
+/// A wrapped `FileDescriptor` or `DispatchIO` and
+/// whether it should beeddsw closed automactially when done.
+internal struct DiskIO {
```

Comment:
I'm not sure I like this direction. These really feel like two types smashed into a hidden enum. I think it will feel better if we create a new type for tracking the dispatch IO and use it where appropriate instead.